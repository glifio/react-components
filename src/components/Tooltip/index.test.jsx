import { render } from '@testing-library/react'
import { Base } from './index.stories'

describe('Tooltip', () => {
  test('storybook example renders', () => {
    const { container } = render(<Base {...Base.args} />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
