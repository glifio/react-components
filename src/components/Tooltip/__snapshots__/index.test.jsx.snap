// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledATag storybook example renders 1`] = `
.c2 {
  width: 2em;
  height: 2em;
  padding: 0;
  line-height: 2em;
  border-radius: 1em;
  border: 1px solid var(--black);
  color: var(--black);
  background: transparent;
  text-align: center;
  cursor: pointer;
}

.c4 {
  display: none;
  position: absolute;
  top: calc(100% + 0.5em);
  left: 50%;
  width: -webkit-max-content;
  width: -moz-max-content;
  width: max-content;
  max-width: 20em;
  -webkit-transform: translateX(-50%);
  -ms-transform: translateX(-50%);
  transform: translateX(-50%);
  padding: 0.5em 0.75em;
  color: var(--black);
  background-color: var(--white);
  border-radius: 4px;
  box-shadow: rgba(0,0,0,0.10) 0px 0.7px 2.2px -8px,rgba(0,0,0,0.04) 0px 1.7px 2.4px,rgba(0,0,0,0.106) 0px 3.1px 8.1px,rgba(0,0,0,0.04) 0px 5.6px 12.1px,rgba(0,0,0,0.045) 0px 4.4px 4.8px,rgba(0,0,0,0.05) 0px 15px 41px;
  z-index: 9;
}

.c0 {
  position: relative;
}

.c0:hover .c1 {
  color: var(--purple-medium);
  border-color: var(--purple-medium);
}

.c0:hover .c3 {
  display: block;
}

<div
  class="c0"
>
  <button
    class="c1 c2"
  >
    ?
  </button>
  <div
    class="c3 c4"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer nec est ultrices, posuere lorem a, convallis quam. Etiam venenatis velit.
  </div>
</div>
`;
